shopfloor/actions/message.py

```diff
@@ -581,10 +581,14 @@ def transfer_complete(self, picking):
             "body": _("Transfer {} complete").format(picking.name),
         }
 
-    def location_content_transfer_item_complete(self, location_dest):
+    def location_content_transfer_item_complete(self, location_src, location_dest):
         return {
             "message_type": "success",
-            "body": _("Content transfer to {} completed").format(location_dest.name),
+            "body": _(
+                "Content line transferred from %(location_name)s to %(location_dest_name)s",
```

Comment:
What about
```suggestion
                "Transferred from %(location_name)s to %(location_dest_name)s",
```
or
```suggestion
                "Product %(product_name) transferred from %(location_name)s to %(location_dest_name)s",
```
and
```suggestion
                "Package %(package_name) transferred from %(location_name)s to %(location_dest_name)s",
```